source/tutorials.rst

```diff
@@ -0,0 +1,122 @@
+Tutorials
+=========
+
+
+Getting started with ``pytest_container``
+-----------------------------------------
+
+.. note::
+
+   In this tutorial we will start with a minimal python project and start using
+   ``pytest_container`` to test its command line interface.
+
+
+We will use `poetry <https://python-poetry.org/>`_ to manage our python
+dependencies in this example and assume that you already have a python project
+with a :file:`pyproject.toml`. If not, run :command:`poetry init` to create a
+new one interactively.
+
+First, we need to add ``pytest_container`` as a development dependency:
+
+.. code-block:: shell-session
+
+   ❯ poetry add --dev pytest_container
+
+
+We also recommend to add `pytest-xdist
+<https://github.com/pytest-dev/pytest-xdist>`_ so that tests can be executed in
+parallel:
+
+.. code-block:: shell-session
+
+   ❯ poetry add --dev pytest-xdist
+
+
+Now we are ready to start writing our tests. Start by creating (if not yet
+present) a directory for your tests, preferably called :file:`tests` or
+:file:`test`. Add an empty :file:`tests/__init__.py` and create the file
+:file:`tests/conftest.py` with the following contents:
+
+.. code-block:: python
+
+   from pytest_container import auto_container_parametrize
+
+
+   def pytest_generate_tests(metafunc):
+       auto_container_parametrize(metafunc)
+
+
+
+The above code snippet ensures that the ``auto_container`` and
+``auto_container_per_test`` fixtures work correctly (more specifically, that
+they are using the images defined in ``CONTAINER_IMAGES``).
+
+
+We can now implement the actual tests. We will create a simple smoke test where
+we install the python wheel of our project inside a container and then check
+that the installed command line utility works as expected. For that, we will
+start out with a :file:`Dockerfile`:
+
+.. code-block:: Dockerfile
+
+   FROM registry.opensuse.org/opensuse/tumbleweed
+
+   RUN zypper -n in python3-pip
+   COPY dist/*whl .
+   RUN pip install *whl
+
+
+Assuming that this :file:`Dockerfile` were in our project's root and our python
+project is built into the :file:`dist/` folder, we would build an image based on
+openSUSE Tumbleweed with our project installed into it.
```

Comment:
```suggestion
Assuming that the :file:`Dockerfile` resides in the project's root and the Python
project is stored in the :file:`dist/` folder, build an image based on
openSUSE Tumbleweed with the project installed into it.
```

This still doesn't make sense to me. Project installed into what (what does _into it_ refer to)? Can project be installed into something?